src/main/scala/org/broadinstitute/dsde/workbench/sam/service/StatusService.scala

```diff
@@ -26,7 +26,7 @@ class StatusService(
   implicit val askTimeout = Timeout(5 seconds)
 
   private val healthMonitor = system.actorOf(HealthMonitor.props(cloudExtensions.allSubSystems + OpenDJ)(checkStatus _))
-  system.scheduler.schedule(initialDelay, pollInterval, healthMonitor, HealthMonitor.CheckAll)
```

Comment:
schedule is deprecated and scheduleAtFixedRate is the recommended, performant replacement